lib/contentful/importer/converters/content_types_structure_creator.rb

```diff
@@ -1,5 +1,5 @@
 module Contentful
-  module Converter
+  module Importer
```

Comment:
Do we want to use the `Contentful::Importer::Converter` namespace here or is this too namespacey?